README.md

```diff
@@ -296,6 +296,33 @@ $menu = (new CliMenuBuilder)
     ->build();
 ```
 
+If you want to use the full width of the terminal, you can grab the terminal object and ask/set it from there like so:
+
+```php
+<?php
+
+use PhpSchool\CliMenu\Builder\CliMenuBuilder;
+
+$menu = ($builder = new CliMenuBuilder)
+    ->setWidth($builder->getTerminal()->getWidth())
+    ->build();
+```
+
+If you want to use the full width of the terminal and apply a margin you will need to perform a little calculation yourself
+since we use a box model similar to CSS. The example below will use the full width of the terminal plus a margin of 2 on the 
```

Comment:
Why not just mention what is included in the width and what not? It should be made clear that the margin is not included.